src/main/java/com/emc/mongoose/storage/driver/kafka/KafkaStorageDriver.java

```diff
@@ -159,13 +166,27 @@ protected String requestNewAuthToken(Credential credential) {
       final I lastPrevItem,
       final int count)
       throws IOException {
-
+    
     if (listWasCalled) {
       throw new EOFException();
     }
 
     val buff = new ArrayList<I>(1);
-    buff.add(itemFactory.getItem(path + prefix, 0, 0));
+
+    if (itemFactory instanceof PathItemFactoryImpl) {
+      Properties properties = new Properties();
```

Comment:
We have a properties instances cache:
https://github.com/emc-mongoose/mongoose-storage-driver-kafka/blob/966c7772279e857af5b076dd07669a318c4247ff/src/main/java/com/emc/mongoose/storage/driver/kafka/KafkaStorageDriver.java#L52